Source/IntegrationTests/LitTests.cs

```diff
@@ -104,7 +107,7 @@ public class LitTests {
           Environment.GetEnvironmentVariable("HOMEDRIVE") + Environment.GetEnvironmentVariable("HOMEPATH"));
         passthroughEnvironmentVariables = passthroughEnvironmentVariables
           .Concat(new[] {
-            "DOTNET_CLI_HOME",
+              "DOTNET_CLI_HOME",
```

Comment:
Please Remove extra space? Verify that there are no tabs as well.

Reply:
Woops, thanks!